code/presets/react-webpack/src/loaders/react-docgen-loader.ts

```diff
@@ -114,7 +114,10 @@ export default async function reactDocgenLoader(
       }
     });
 
-    const map = magicString.generateMap({ hires: true });
+    const map = magicString.generateMap({
```

Comment:
I don't think it's that crucial. It now behaves like react-docgen in the vite builder. Usually, when using sourcemaps, it's enough to point to the line instead of the exact char position when the original files are in place. And since this wasn't ever working, I would start with the more performant variant and improve the source map detail level on request.

Reply:
Sure thing, let's just not forget that this setting was there in the first place!